README.md

```diff
@@ -1,6 +1,14 @@
 # Terra UI
 Web user interface for the Terra platform.
 
+------------------------
+### Links:
+[Support](https://support.terra.bio/hc/en-us)  
+[File an issue](mailto:terra-support@broadinstitute.zendesk.com)  
+[Board and backlog](https://broadworkbench.atlassian.net/projects/SATURN/issues?filter=allopenissues&orderby=status%20DESC)
```

Comment:
```suggestion
[Board and backlog](https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=41)
```
Jira link was highlighting a specific issue, we want to link to the board

Reply:
Ah, I wanted to link to all issues, so that if someone creates an issue, they can see it in the list. It insists on highlighting the first issue in the list.

Reply:
Link to the backlog? https://broadworkbench.atlassian.net/secure/RapidBoard.jspa?rapidView=41&projectKey=SATURN&view=planning.nodetail

Reply:
Yeah, but that hides issues in progress...